src/backup/BackupPhrase.tsx

```diff
@@ -97,24 +96,23 @@ class BackupPhrase extends React.Component<Props, State> {
 
   onPressContinue = () => {
     ValoraAnalytics.track(OnboardingEvents.backup_continue)
-    navigate(Screens.BackupQuiz, { navigatedFromSettings: this.navigatedFromSettings() })
+    navigate(Screens.BackupQuiz, { settingsScreen: this.settingsScreen() })
   }
 
-  navigatedFromSettings = () => {
-    return this.props.route.params?.navigatedFromSettings ?? false
+  settingsScreen = () => {
+    return this.props.route.params?.settingsScreen ?? undefined
   }
 
   render() {
     const { t, backupCompleted } = this.props
     const { mnemonic, isConfirmChecked } = this.state
-    const navigatedFromSettings = this.navigatedFromSettings()
     return (
       <SafeAreaView style={styles.container}>
         <CustomHeader
           style={{ paddingHorizontal: variables.contentPadding }}
           left={
-            navigatedFromSettings ? (
-              <CancelButton style={styles.cancelButton} />
```

Comment:
seems like this was just navigating back previously ? could we retain the same behavior here?